data/json/monsters.json

```diff
@@ -1773,7 +1773,11 @@
         ["GRAB", 7]
     ],
     "description":"Still wearing the tattered remnants of improvised armor and weaponry, it is plain to see that this zombie was once a survivor like you.",
-    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"]
+    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"],
+    "upgrades":{
+        "into": "mon_zombie_survivor_elite",
```

Comment:
Nope
We don't have mechanics of limiting drops to old forms.
Unless it learns how to sew, weld and scavenge while in zombie form, it won't produce the items that elites drop.